### ISSUE TITLE
bundle install fails due to mimemagic dependency constraint

### ISSUE THREAD

**Issue by davidsnyder-jask** [salsify/avro-schema-registry#319]
We're seeing 

```
[2021-04-07T18:11:29.754Z] Step 5/15 : RUN gem install bundler --no-document && bundle install --jobs 20 --retry 5
[2021-04-07T18:11:29.754Z]  ---> Running in 48db59532dc3
[2021-04-07T18:11:30.011Z] Successfully installed bundler-2.2.15
[2021-04-07T18:11:30.011Z] 1 gem installed
[2021-04-07T18:11:32.717Z] Fetching gem metadata from https://rubygems.org/.........
[2021-04-07T18:11:32.717Z] Your bundle is locked to mimemagic (0.3.3), but that version could not be found
[2021-04-07T18:11:32.717Z] in any of the sources listed in your Gemfile. If you haven't changed sources,
[2021-04-07T18:11:32.717Z] that means the author of mimemagic (0.3.3) has removed it. You'll need to update
[2021-04-07T18:11:32.717Z] your bundle to a version other than mimemagic (0.3.3) that hasn't been removed
[2021-04-07T18:11:32.717Z] in order to install.
```

when building the Dockerfile. Seems like you need to bump the `mimemagic` dependency to `0.3.6`. https://github.com/mimemagicrb/mimemagic/issues/98

**Comment by davidsnyder-jask** [comment]
To be clear it's showing `0.3.3` because we're pegged to an older version of this repo: https://github.com/salsify/avro-schema-registry/commit/034c9581db09f84f362dcb7698f9f860b74a6fc8 but I think you will hit the same issue with `master`

**Comment by jturkel** [comment]
Fixed by #317
